### PR TITLE
[rc] arcticdb v3.0.0rc0

### PR DESCRIPTION
**Pull request by phoebusm**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by phoebusm** [comment]
RC is missed in the `v3.0.0` release. This is a retro rc build for it.

**Comment by jjerphan** [comment]
We need to create a dedicated `3.0.0rc` tag for the release candidate.

Could you pin `msgpack_c`, as done on main with: https://github.com/conda-forge/arcticdb-feedstock/commit/31a5f168479551258913aa5f7d6ec8a89d36c05d and rerender?

**Review by jjerphan — COMMENTED** [review]
We need to create a dedicated `3.0.0rc` tag for the release candidate.

Could you pin `msgpack_c`, as done on `main` with: https://github.com/conda-forge/arcticdb-feedstock/commit/31a5f168479551258913aa5f7d6ec8a89d36c05d and rerender?

**Comment by phoebusm** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: a77c3f9579a9e4dc907adb799c98f6aa4e2fcfb0d470e8067bd438f77d81acd6
+  sha256: d6af45647e1027f81dfac9979042607f432538eec43c8ccac2efa2b8be6f8a03
 
   patches:
```
Comment (jjerphan) [inline recipe/meta.yaml:12]: This section (which is now uneeded), has to be removed.

File: recipe/meta.yaml
Comment (jjerphan) [inline recipe/meta.yaml:1]: `packaging` also needs to be added as a dependencies [as done in #49 there](https://github.com/conda-forge/arcticdb-feedstock/pull/49/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR100).

**Review by jjerphan — APPROVED** [review]
LGTM given :green_circle: CI checks.


Edit: I change the title of the PR for consistency.

**Comment by h-vetinari** [comment]
I don't know exactly how it happened, but someone or something managed to trigger the CI for this PR a few times, and now conda-forge is running out of agents:
image

In case you've been eagerly clicking the re-run button (e.g. because fetches currently fail due to github issues), then please stop. You can only restart through the GH UI _after_ the job has completely finished.

**Comment by jjerphan** [comment]
Thank you for giving precision.

I am the culprit. :hand: 

To give precision, I reran the jobs which were failing due [to the GitHub issues](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=775033&view=logs&j=bb1c2637-64c6-57bd-9ea6-93823b2df951&t=55f066f8-9d52-54c8-1ae3-a2d9002ff304&l=593) via [this pannel](https://github.com/conda-forge/arcticdb-feedstock/runs/16501045918) and I did not think this would have had consequences for conda-forge's infrastructure. Now I know, and I will not do it in the future.

**Pull request merged**